src/components/dragelement/unhover.js

```diff
@@ -34,7 +34,7 @@ unhover.raw = function raw(gd, evt) {
     var oldhoverdata = gd._hoverdata;
 
     if(!evt) evt = {};
-    if(evt.target &&
+    if(evt.target && !gd._dragged &&
```

Comment:
I was wondering if in addition to `gd._dragged` we should check for `gd._editing` here?

However the following codepen titled `after` appears to be correct when editing the title positioned at center.
[Before](https://codepen.io/MojtabaSamimi/pen/yLaWbMW?editors=0010) vs [After](https://codepen.io/MojtabaSamimi/pen/BaLeRZV?editors=0010)

So I think there is no need for that.

Reply:
Good point. I'll check that as well.